lib/routes/cags/namespace.ts

```diff
@@ -0,0 +1,22 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'CAGS Edu',
+    url: 'https://edu.cags.ac.cn',
+    description: `
+    :::tip
+    中国地质科学院研究生院网站的部分通知公告可以通过以下 RSS 路由订阅：
+
+    -   通知公告：\`/tzgg\`
+    -   要闻简讯：\`/ywjx\`
+    -   博士生招生：\`/zs-bss\`
+    -   硕士生招生：\`/zs-sss\`
+
+    可以根据不同的需求来订阅对应页面的最新公告信息。
+    :::
```

Comment:
Indent with either 0 or 2 spaces or the markdown content won't be rendered properly.